contents/julia_101.md

```diff
@@ -1,40 +1,438 @@
 # Julia 101 {#sec:julia_101}
 
-We should include a nice warning box here saying:
-
 > **_NOTE:_**
 > In this chapter we will cover the basic of Julia as a programming language.
 > Please note that this is not *strictly necessary* for you to use Julia as a tool of data manipulation and data visualization.
 > Having a basic understanding of Julia will definitely make you more *effective* and *efficient* in using Julia.
 > However, if you prefer to get started straight away, you can jump to [@sec:dataframes] to learn about tabular data with DataFrames.jl.
 
+This is going to be a very brief and *not* in-depth overview of Julia language.
+If you are already familiar and confortable with other programming languages, we highly encourage you to read [Julia's documentation](https://docs.julialang.org/).
+It is a thoroughly deep dive into Julia.
+It covers all the basics and corner cases, but it can be cumbersome.
```

Comment:
```suggestion
It covers all the basics and corner cases, which can be cumbersome.
```

Reply:
I don't know, which can be cumbersome leaves no room for doubt. One thing implies the other. But leaves some room for doubt. It "could" be...